src/main/java/com/cloudbees/plugins/credentials/SystemCredentialsProvider.java

```diff
@@ -629,7 +629,7 @@ public CredentialsStore getStore() {
         @Override
         public String getIconFileName() {
             return isVisible()
-                    ? "symbol-jenkins"
+                    ? "/jenkins/plugin/credentials/images/system-store.svg"
```

Comment:
I think this will only work if you have Jenkins deployed in the `jenkins` context.

also this should be using the [icon framework](https://github.com/jenkinsci/credentials-plugin/blob/de334b12145ca0c1eeb19689246fdc58fa3c0ca7/src/main/java/com/cloudbees/plugins/credentials/ContextMenuIconUtils.java#L180-L203) to lookup the icon.

so I think you just need

```suggestion
                    ? "system-store"
```

Reply:
icon class is speficied in getIconClass not Filename.  this method appears to be totally unused if the class is present and represents an icon.